WcaOnRails/app/models/user.rb

```diff
@@ -1116,7 +1116,7 @@ def url
     include: ["avatar", "teams"],
   }.freeze
 
-  def serializable_hash(options = nil)
+  def serializable_hash(options = nil, overwrite_default = false)
```

Comment:
Sorry but veto.
We will not override the way Rails behaves -- fighting with Rails conventions is a bad idea because Rails always wins in the end.

I will submit another comment showing you how to achieve your desired behavior.